makeflow/src/makeflow.c

```diff
@@ -1370,7 +1370,7 @@ int main(int argc, char *argv[])
 				}
 				break;
 			case LONG_OPT_DISABLE_BATCH_CACHE:
-				cache_mode = 0;
+				cache_mode = 1;
```

Comment:
Also, note that for WQ, WORK_QUEUE_NOCACHE is 0, while CACHE is 1. Not that it matters, as it is an implementation detail of the constant, but it is better to keep this consistent.